### PR TITLE
Team 5 Requesting Signoff for Safety System

### PR DESCRIPTION
**Pull request by MTurpeau**

### CODE REVIEW

File: Documentation/Signoffs/Safety_System.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Safety System
+
+# Subsystem Function:
+The sole purpose of this system is to secure the safety of both the users and the operators of the device. The primary functions of the system come from cabinet restrictions, modified chassis, and informational stickers. Another focus of this system is to meet all the requirement of having a piece of electrical machinery. These standards are mostly taken from OSHA's Electrical General Requirements page.
+
+
+
+# Constraints:
+
++ The system must protect users from possible self-harm
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Safety_System.md]: Self-harm is not relevant here.

File: Documentation/Signoffs/Safety_System.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+# Safety System
+
+# Subsystem Function:
+The sole purpose of this system is to secure the safety of both the users and the operators of the device. The primary functions of the system come from cabinet restrictions, modified chassis, and informational stickers. Another focus of this system is to meet all the requirement of having a piece of electrical machinery. These standards are mostly taken from OSHA's Electrical General Requirements page.
+
+
+
+# Constraints:
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Safety_System.md]: Where are these standards coming from and why are they here? They aren't relevant.

**Review by JesseTNRoberts — COMMENTED** [review]
Update based on in person discussion

**Review by JesseTNRoberts — COMMENTED** [review]
This is not redundant. Your PLC is not redundant and is part of the control system. This has to be separate and redundant. 

I recommend using the output of these sensors to directly control a relay.

**Comment by MTurpeau** [comment]
Wanted your thoughts on the work so far. And if there is any advice on what to use to make a forced stop of the power to the plc. in most demonstrations, the relay/plc has its own power supply that responds to the relay output. Also, there been a common concern for the budget, and any advice on how to reduce the cost of the system would be greatly appreciated. Thank you

**Review by JesseTNRoberts — COMMENTED** [review]
There's no need to use a safety rated relay. It is sufficient to two regular relays (2 gives the required redundancy). Other than that, it looks like you are on the right track. The pink wire of the last switch will control one relay and the gray will control the other. Power to the motor should go through both relays. That way, if a door opens, the motor cannot turn.

**Comment by MTurpeau** [comment]
I updated the system to match our earlier discussions. While this design still uses the non-contact switches, I have placed the schematic of the system that uses 2 separate induction circuits below the full system schematic. Please let me know if I should go ahead and switch out the hypothetical system with the actual system.

**Review by JesseTNRoberts — APPROVED** [review]
I'm approving. However, the schematic for using the regular proxes won't work. For that one you will have to have a method of creating a logical AND operation to AND each of the three sets. The safety sensors implement this for you, but the regular proxes don't.

**Pull request merged**